src/encryption/ec.ts

```diff
@@ -119,13 +119,13 @@ export function encryptECIES(publicKey: string, content: string | Buffer): Ciphe
   )
 
   const macData = Buffer.concat([initializationVector,
-                                 Buffer.from(ephemeralPK.encodeCompressed()),
+                                 Buffer.from(ephemeralPK.encodeCompressed('array') as any),
```

Comment:
@friedger Are you getting type errors here as well? I had to do this to be able to make a build. Problem is that I'm now getting `Unknown point format` encryption errors in the browser when trying to use `getFile`. Do you have any clue what's going on here? 🤔

Reply:
Type error was that `encodeCompressed` expects one argument and the `as any` is necessary to avoid the `BN can't be cast to a string` type error. Which is weird because I don't think `Buffer.from` expects a string 😕

Reply:
The typings in the `@types/elliptic` package are incomplete and incorrect in some areas. In this case, I don't think `encodeCompressed` even returns a `BN`. If you merge the develop branch into this PR, it is updated to use a different function with more accurate typings. Something like `Buffer.from(ephemeralPK.encode('array', true) as Buffer)`.

One of my goals is to move us away from the `elliptic` lib all together at some point. For now I recommend going off the actual `elliptic` lib source and forcing the types as necessary.